compiler/src/dotty/tools/dotc/transform/Erasure.scala

```diff
@@ -29,6 +29,10 @@ import TypeUtils._
 import ExplicitOuter._
 import core.Mode
 import core.PhantomErasure
+import dotty.tools.dotc.core.Annotations.Annotation
+import dotty.tools.dotc.core.classfile.ClassfileConstants
+import dotty.tools.dotc.core.TypeApplications.TypeParamInfo
+import java.lang.StringBuilder
 
 class Erasure extends Phase with DenotTransformer { thisTransformer =>
 
```

Comment:
Erasure is already rather large as is, and therefore people are scared to touch it. We should try to signature generation in a separate sourcefile. It could be integrated as an object in erasure similarly to how Bridges.scala moves out bridge generation from erasure.